docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,201 @@
+======================
+HTTP client properties
+======================
+
+HTTP client properties allow you to configure the connection from Trino to
+external services using HTTP.
+
+The following properties can be used after adding the specific prefix to the
+property. For example, for :doc:`/security/oauth2`, you can enable HTTP for
+interactions with the external OAuth 2.0 provider by adding the prefix
+``oauth2-jwk`` to the ``http-client.http.enabled``, and setting the value to
+``true``:
+
+.. code-block::
+
+    oauth2-jwk.http-client.http2.enabled=true
+
+The following prefixes are supported:
+
+* ``oauth2-jwk`` for :doc:`/security/oauth2`
+* ``jwk`` for :doc:`/security/jwt`
+
+General properties
+------------------
+
+``http-client.https.excluded-cipher``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+
+A comma-separated list of regexes for the names of cipher algorithms to exclude.
+
+``http-client.https.included-cipher``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+
+A comma-separated list of regexes for the names of the cipher algorithms to use.
+
+``http-client.log.compression-enabled``
```

Comment:
Let's move all of the `http-client.log` properties to a new "Request logging" section.